src/Workspaces/CSharpTest/CodeGeneration/SyntaxGeneratorTests.cs

```diff
@@ -223,6 +223,11 @@ public void TestTypeExpressions()
             VerifySyntax<TypeSyntax>(_g.ArrayTypeExpression(_g.ArrayTypeExpression(_g.IdentifierName("x"))), "x[][]");
             VerifySyntax<TypeSyntax>(_g.NullableTypeExpression(_g.IdentifierName("x")), "x?");
             VerifySyntax<TypeSyntax>(_g.NullableTypeExpression(_g.NullableTypeExpression(_g.IdentifierName("x"))), "x?");
+
+            VerifySyntax<TypeSyntax>(_g.TupleTypeExpression(_g.TupleElementExpression(_g.IdentifierName("x"), "y") ), "(x y)");
```

Comment:
:question: Is it possible to construct the `ITypeSymbol` for a tuple type, and just make the existing `TypeExpression` construct do the right thing? That would eliminate the need for `TupleTypeExpression` and `TupleElementExpression`.